google/cloud/publicca/README.md

```diff
@@ -0,0 +1,60 @@
+# Public Certificate Authority API C++ Client Library
+
+This directory contains an idiomatic C++ client library for the \[Public
+Certificate Authority API\]\[cloud-service-docs\].
```

Comment:
Hmm.. what happened here?

The scaffolding looks fine to me:
https://github.com/googleapis/google-cloud-cpp/blob/f992e93a50b187b096c6148c8603cf732025c2fa/generator/internal/scaffold_generator.cc#L349-L350

But maybe it does not work with `checkers` anymore?